### PR TITLE
Arabic

### PR DESCRIPTION
**Pull request by operationairstrike**

### CODE REVIEW

**Review by Zaid-maker — COMMENTED** [review]
Please add a single language translation at a time with proper Title and descrition of the PR

**Comment by D3-3109** [comment]
Please do not use Google Translate to translate English into Asian or Middle Eastern languages , this could make lots of mistakes.

**Comment by Zaid-maker** [comment]
> Please do not use Google Translate to translate English into Asian or Middle Eastern languages , this could make lots of mistakes.

Yup you are correct!

**Review by D3-3109 — CHANGES_REQUESTED** [review]

File: frontend/src/lang/jp-JP.json [hunk 1]
```diff
@@ -0,0 +1,94 @@
+{
+  "languageName": "英語",
```
[the remark this answers is not in the document]
Comment (louislam) [reply frontend/src/lang/jp-JP.json:2]: Yes, it is incorrect, I think I just close this pr.

**Pull request closed without merging**